config/application.rb

```diff
@@ -14,7 +14,7 @@ class Application < Rails::Application
 
     # Set Time.zone default to the specified zone and make Active Record auto-convert to this zone.
     # Run "rake -D time" for a list of tasks for finding time zone names. Default is UTC.
-    # config.time_zone = 'Central Time (US & Canada)'
+    config.time_zone = 'Central Time (US & Canada)'
 
```

Comment:
You don't have to set this. My other comment refers to the fact that Time.now doesn't take into accountof the timezone configures in rails